src/app/nextversion/nextversion.go

```diff
@@ -129,6 +130,13 @@ func NextVersion(cCtx *cli.Context) error {
 	bmpr.DependencyCap = dependencyCap
 
 	next, err := bmpr.BumpSource(versionSrc)
```

Comment:
I think we were previously ignoring this error. Some of the thousand linters should have caught this, right? Or am I missing something?

Reply:
I was missing something, it is handled in the switch statement. I added a comment to clarify this.